jobs/roleNotifier.js

```diff
@@ -0,0 +1,23 @@
+import { Role } from "../models";
+import { roleNotifierMail } from "./templates";
+
+export default async transporter => {
+
+    if(!transporter)
+        return;
+
+    const inactiveRoles = await Role.find({newlyAdded: true});
+
+    if(inactiveRoles.length === 0)
+        return;
+
+    await Role.updateMany({}, {$unset: {newlyAdded: true}});
```

Comment:
It could be updated manually by admin. We don't want to display every role even these with inappropriate words

Reply:
But we are just removing the no-longer-needed newlyAdded property, the isActive property is still false at this moment

Reply:
I think we have to get report on email with new roles (for example weekly) and decide to activate chosen one